### PR TITLE
Upgrades to the LA Metro scraper

### PR DESCRIPTION
**Pull request by fgregg**
This PR makes changes to the LA Metro scraper to try to handle in the data layer issues that had been handled in Metro's Councilmatic view code. Most significantly:

1. trying to find at attach minutes to meetings
2. attaching family name to person (which we'll use for sorting)
3. agenda_numbers to extras for sorting

### CODE REVIEW

**Comment by hancush** [comment]
Also https://github.com/datamade/la-metro-councilmatic/pull/510#discussion_r350354676

**Comment by hancush** [comment]
@fgregg Does this PR supersede #291?

File: lametro/people.py [hunk 1]
```diff
@@ -98,6 +100,9 @@ def scrape(self):
             person_api_url, person_web_url = source_urls
             p.add_source(person_api_url, note='api')
             p.add_source(person_web_url, note='web')
+            p.family_name = term['OfficeRecordLastName']
+            p.given_name = term['OfficeRecordFirstName']
+
```
Comment (hancush) [inline lametro/people.py]: @fgregg What's going on here? This seems to be duplicating (or overriding) attributes set in L68-69?

Comment (fgregg) [reply lametro/people.py]: you are right. this can be removed.

**Comment by fgregg** [comment]
> @fgregg Does this PR supersede #291?

@hancush, yes.

**Pull request merged**